multihost/db-postgresql-with-custom-certificates/main.fmf

```diff
@@ -0,0 +1,37 @@
+summary: Tests basic keylime attestation scenario on multiple hosts with custom certificates and postgresql database
+description: |
+ Running services on different systems - verifier, registrar, agent+tenant.
```

Comment:
Would be worth updating, verifier+registrar is running together. Also the description should mention that verifier and registrar are configured using a keylime_server role.